packages/boa/tools/install-python.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+const { run, PLATFORM, ARCH } = require('./utils');
+const fs = require('fs');
+const path = require('path');
+
+const CONDA_DOWNLOAD_PREFIX = 'https://repo.anaconda.com/miniconda';
+let CONDA_DOWNLOAD_NAME = 'Miniconda3-latest';
```

Comment:
By general convention, only constants are capitalized？

Reply:
IMO, the consistence with the CONDA_DOWNLOAD_PREFIX might be more important here :)